spec/cc/engine/csslint_spec.rb

```diff
@@ -34,7 +34,11 @@ module Engine
 
         describe "with include_paths" do
           let(:engine_config) {
-            {"include_paths" => %w(included.css included_dir/ config.yml)}
+            {
+              "config" => {
+                "include_paths" => %w(included.css included_dir/ config.yml)
```

Comment:
Should `include_paths` be under `config`? I thought `config` was for the user-provided configuration?